python-connectors/pi-system_event-frames-search/connector.py

```diff
@@ -149,7 +151,7 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
                         yield event_frame_copy
                         limit.add_record()
                         if limit.is_reached():
```

Comment:
I think we can/should probably remove this call to add_record() too?

Reply:
indeed ! [a0f8bb8](https://github.com/dataiku/dss-plugin-osisoft/pull/20/commits/a0f8bb8cda341f300c1be2729618a5913fa3625f)

Reply:
👍